src/main/archetype/ui.content/pom.xml

```diff
@@ -55,8 +48,20 @@
                 <artifactId>filevault-package-maven-plugin</artifactId>
                 <extensions>true</extensions>
                 <configuration>
-                    <acHandling>merge_preserve</acHandling>
-                    <group>${packageGroup}</group>
+                    <group>${groupId}</group>
+                    <name>${appsFolderName}.ui.content</name>
```

Comment:
Curious - whats the rationale for this being named with `${appsFolderName}` vs `${artifactId}` which is used in ui.apps? If anything it seems ui.apps should use the appFolderName (if anything does)...

Reply:
Fixed in https://github.com/adobe/aem-project-archetype/pull/256/commits/b4b94d13dfd5506e5e2d5d617b41fa3de66fb10a#diff-e47625583bde4abbefd820c562c735cdR52 and https://github.com/adobe/aem-project-archetype/pull/256/commits/b4b94d13dfd5506e5e2d5d617b41fa3de66fb10a#diff-9a5eca8b7f42611af149eeabb4c3686eR53